src/commands/index.ts

```diff
@@ -42,8 +43,9 @@ const commands = [
   kubeconfigCommand,
 ];
 
-const buildArgv = () => {
-  const argv = yargs(hideBin(process.argv));
+const buildArgv = async () => {
+  const { version } = await P0_VERSION_INFO;
+  const argv = yargs(hideBin(process.argv)).version(version);
```

Comment:
This manually tells yargs what the version number is. This was broken in the standalone CLI, because yargs reads this info from the package.json file, which doesn't exist in a standalone installation.